modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/model/impl/CPSpecificationOptionImpl.java

```diff
@@ -75,13 +77,19 @@ public long getListTypeDefinitionsCount() throws PortalException {
 
 	@Override
 	public List<ListTypeEntry> getListTypeEntries() {
+		long[] listTypeDefinitionIds = TransformUtil.transformToLongArray(
```

Comment:
Move this method where it is called, without using Util classes and remove this one